templates/submission/submissionMetadataFormTitleFields.tpl

```diff
@@ -21,6 +29,6 @@
 	{/fbvFormSection}
 	{fbvFormSection description="common.subtitle.tip"}{/fbvFormSection}
 	{fbvFormSection title="common.abstract" for="abstract" required=$abstractsRequired}
-		{fbvElement type="textarea" multilingual=true name="abstract" id="abstract" value=$abstract rich="extended" readonly=$readOnly}
+		{fbvElement type="textarea" multilingual=true name="abstract" id="abstract" value=$abstract rich=$rich readonly=$readOnly}
```

Comment:
Yes, that was intentional. For the old revisions, I deactivate all input fields - and I realized that deactivating a textarea is only possible when it's not in the "rich" mode. That means that I have to turn off the "rich" mode first when I want to make it read-only.